##[pylyzer] failed /home/dani/advent-of-code-2024/day13/venv/lib/python3.12/site-packages/scipy/stats/_relative_risk.py 1734628983 9571
.operator = pyimport "<failure>"
.<failure> = pyimport "<failure>"

.___v_desugar_1 = pyimport "<failure>"

.dataclass: Never
.np = pyimport "__init__"
.__init__ = pyimport "__init__"

.___v_desugar_2 = pyimport "__init__"

.ndtri: Never
.___v_desugar_3 = pyimport "_common"
._common = pyimport "_common"
.ConfidenceInterval: Never
._validate_int: |Type_96766 <: Structural({.__lt__ = (self: Never, ?R) -> Bool}), R: Type|(n: Type_96766, bound: R, name: Obj) -> Type_96766
.RelativeRiskResult: ClassType
.RelativeRiskResult.control_cases: Int
.RelativeRiskResult.exposed_cases: Int
.RelativeRiskResult.relative_risk: Float
.RelativeRiskResult.exposed_total: Int
.RelativeRiskResult.control_total: Int
.RelativeRiskResult.__call__: () -> _relative_risk.RelativeRiskResult
.RelativeRiskResult.confidence_interval: (self: _relative_risk.RelativeRiskResult, confidence_level: Nat := Nat) -> Never

.relative_risk: (exposed_cases: Never, exposed_total: Never, control_cases: Never, control_total: Never) -> Never
